client/Packages/com.beamable.server/Editor/Usam/Editor/Signposts/BeamServiceSignpost.cs

```diff
@@ -14,7 +15,7 @@ public class BeamServiceSignpost : ISignpostData
 		public string name;
 		public string assetProjectPath = "";
 
-		public string[] assemblyReferences;
+		public AssemblyDefinitionAsset[] assemblyReferences;

```

Comment:
out of curiosity- what does this look like in the actual file content?

Reply:
![image](https://github.com/beamable/BeamableProduct/assets/141041673/80c03ce3-b7e4-420b-a314-398d930160d8)
`AStar` is an assembly reference of a A* path finding asset that I downloaded from the asset store.